src/sss_client/pam_sss.c

```diff
@@ -50,6 +50,8 @@
 #include "util/authtok-utils.h"
 #include "util/dlinklist.h"
 
+void sss_erase_mem_securely(void *p, size_t size);  /* from memory_erase.c */
```

Comment:
This function is declared in the `util.h` header file. Redeclaring it here is not a good idea.

Reply:
sss_client code doesn't include 'util.h' (for a reason - to not pull any dependencies like 'talloc' etc)

Reply:
I do not have strong opinion on this, but what about moving the declaration to `memory_erase.h`?

Reply:
I can add it if you'd like but I personally don't like adding headers for the sake of one function declaration.

Reply:
I like Tomas' idea. This new header file can be included from `util.h` so that no other change is required, and included directly from the C files that use only this function.